server/src/main/java/com/objectcomputing/checkins/services/questions/QuestionController.java

```diff
@@ -80,26 +80,29 @@ public Question getById(UUID questionid) {
 
     }
 
-    //  I know these are commented out - I added them before splitting the story and will
-    // need them in the future - please ignore for now
-//    /**
-//     * Update the pending status of a skill.
-//     * @param question
-//     * @return
-//     */
-//    @Put("/")
-//    public HttpResponse<?> update(@Body @Valid Question question) {
-//
-//        if(null != question.getQuestionid()) {
-//            Question updatedQuestion = questionService.update(question);
-//            return HttpResponse
-//                    .ok()
-//                    .headers(headers -> headers.location(location(updatedQuestion.getQuestionid())))
-//                    .body(updatedQuestion);
-//        }
-//
-//        return HttpResponse.badRequest();
-//    }
+    /**
+     * Update the text of a question.
+     * @param question
+     * @return
+     */
+    @Put("/")
+    public HttpResponse<?> update(@Body @Valid Question question) {
+
+        if(question.getQuestionid() != null) {
+            Question updatedQuestion = questionService.update(question);
+            if (updatedQuestion != null) {
+                return HttpResponse
+                        .ok()
+                        .headers(headers -> headers.location(location(updatedQuestion.getQuestionid())))
+                        .body(updatedQuestion);
+            } else {
+                return HttpResponse.badRequest();
+            }
+        }
+
+        return HttpResponse.badRequest();
+    }
+
 
     protected URI location(UUID uuid) {
         return URI.create("/questions/" + uuid);
```

Comment:
This should probably be "/services/questions" now since we changed the base url.

Reply:
good catch! got it done